share/autostart

```diff
@@ -6,6 +6,8 @@ hc() {
     herbstclient "$@"
 }
 
+if [ -z $TERMINAL ]; then export TERMINAL="xterm"; fi
```

Comment:
ok. I thought if someone is wondering, why there spawns a xterm and if he realizes that the env var is set, he may try to set $TERMINAL to his fancy terminal emulator and then it's working as he thought it might work.